### PR TITLE
setValue script for finalizing winning governance slates

### PR DESCRIPTION
**Pull request by kangarang**
previously, we could not see the finalized statuses of the governance slates in the ui

i added a mapping `requestProposals` to `ParameterStore`'s state to more-easily keep track of `requestID`-`proposalID` pairs. otherwise, we'd have to go through all the `ProposalCreated` events. however i'm open to other ideas as to how to map `requestID`s to `proposalID`s

### CODE REVIEW

**Comment by akuanti** [comment]
Any contract changes need to be in a separate PR with `audit-fixes` as the base.

I think this solution can break if there is a Gatekeeper upgrade that changes the `requestIDs` before this gets executed. Are we not storing the `proposalIDs` somewhere as `createProposal()` gets called?

**Comment by kangarang** [comment]
let's not go with the current solution of adding onchain state. im going to solve this by adding a table the db

**Comment by kangarang** [comment]
rewrote this using the requests table data

dep: #177

File: api/scripts/setValue.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  let winningSlate;
+  try {
+  // get the winning slate of the previous epoch
+    winningSlate = await gatekeeper.functions.getWinningSlate(epochNumber.sub(1), psAddress);
+  } catch (error) {
+    console.error(error);
+  }
+
+  // get the winning slate requests
+  const slateRequests = await gatekeeper.functions.slateRequests(winningSlate);
```
Comment (akuanti) [inline api/scripts/setValue.js]: If the call to `getWinningSlate()` fails, then `winningSlate` will be undefined. The script should probably exit when that happens.

File: api/scripts/setValue.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  contractABIs: { ParameterStore },
+} = require('../../packages/panvala-utils');
+
+const { getContracts } = require('../utils/eth');
+const { Request } = require('../models');
+
+run();
+
+async function run() {
+  const { signer, gatekeeper } = getContracts();
```
Comment (akuanti) [inline api/scripts/setValue.js]: Should this use a mnemonic? It says `Error: unknown account #0` when I try to use an `RPC_ENDPOINT` that is an Infura URL.

Comment (kangarang) [reply api/scripts/setValue.js]: ah yes, that is what should be

**Pull request merged**